src/CommandPolykey.ts

```diff
@@ -46,6 +46,7 @@ class CommandPolykey extends commander.Command {
     this.addOption(binOptions.passwordFile);
     this.addOption(binOptions.format);
     this.addOption(binOptions.verbose);
+    this.addOption(binOptions.divertSensitiveFile);
   }
```

Comment:
This isn't the place to add `divertSensitiveFile` as an option. Any options added here are global to all commands. You need to add it to commands that specifically need it. That being the bootstrap and start command.